intl.emu

```diff
@@ -171,10 +170,8 @@
         <emu-alg>
           1. Let _p_ be _maxPrecision_.
           1. <ins>If _x_ is a Decimal128 value, then</ins>
-            1. <ins>Let _v_ be cohort(_x_).</ins>
-            1. <ins>Let _e_ be quantum(_x_).</ins>
-            1. <ins>Assert: _e_ ≤ 0.</ins>
-            1. <ins>Let _n_ be _v_ × 10<sup>-_e_</sup>.</ins>
+            1. <ins>Let _e_ be the smallest integer such that _x_ × 10<sup>-_e_</sup> is an integer.</ins>
```

Comment:
Can we have 0D here?

Reply:
> Can we have 0D here?

No, we can't. That's a good spot. I've moved up the special handling for 0 in the next if block and extended it to handle 0D, too.